src/app/pages/AccountDetailsPage/ContractCodeCard.tsx

```diff
@@ -12,16 +12,18 @@ import { ScrollableDataDisplay } from '../../components/ScrollableDataDisplay'
 import Box from '@mui/material/Box'
 import { CopyToClipboard } from '../../components/CopyToClipboard'
 import { useScreenSize } from '../../hooks/useScreensize'
+import { base64ToHex } from '../../utils/helpers'
 
 export const contractCodeContainerId = 'code'
 
-const CodeDisplay: FC<{ code: string | undefined; label: string; extraTopPadding?: boolean }> = ({
-  code,
+const CodeDisplay: FC<{ rawData: string | undefined; label: string; extraTopPadding?: boolean }> = ({
+  rawData,
   label,
   extraTopPadding,
 }) => {
   const { t } = useTranslation()
   const { isMobile } = useScreenSize()
+  const code = rawData === undefined ? undefined : base64ToHex(rawData)
```

Comment:
I think also add `0x` prefix

Reply:
OK, updated the helper function to do so.